src/examples/java/com/amazonaws/crypto/examples/cryptomaterialsmanager/custom/AlgorithmSuiteEnforcement.java

```diff
@@ -65,8 +59,8 @@
 
         private final CryptoMaterialsManager cmm;
         private final Set<CryptoAlgorithm> ALLOWED_ALGORITHM_SUITES = Collections.unmodifiableSet(new HashSet<>(Arrays.asList(
-                CryptoAlgorithm.ALG_AES_256_GCM_IV12_TAG16_HKDF_SHA384_ECDSA_P384,  // the default algorithm suite
-                CryptoAlgorithm.ALG_AES_256_GCM_IV12_TAG16_HKDF_SHA256)));          // the recommended unsigned algorithm suite
+                CryptoAlgorithm.ALG_AES_256_GCM_IV12_TAG16_HKDF_SHA384_ECDSA_P384,  // the default non-committing algorithm suite
+                CryptoAlgorithm.ALG_AES_256_GCM_IV12_TAG16_HKDF_SHA256)));          // the recommended unsigned non-committing algorithm suite
```

Comment:
Should this be updated to use committing suites?

Reply:
This actually doesn't really work with committing suites. The examples is to say "we want to limit the suites you can use to these two, one signed and one unsigned". The issue is that (for commitment) we only have 1 signed and 1 unsigned suite (I believe). 

So I can update it, but then the example is kind of meaningless (right now) since nothing else is available anyways. Opinions?

Reply:
This is fine to leave for now. Worth revisiting when we go over the examples as a whole though, it might be that this example is now less valuable with commitment policies, or maybe we'd want to update it to restrict to only one algorithm suite.